src/resmom/start_exec.c

```diff
@@ -2416,10 +2468,7 @@ get_new_exec_vnode_host_schedselect(job *pjob, char *msg, size_t msg_size)
 	char	*r_buf = NULL;
 	int	rc = 0;
 
-	/* get exec_vnode */
-	/* don't close pipes */
-
-	/* get exec_vnode */ 
+	/* get exec_vnode don't close pipes */
```

Comment:
You might want to explain this a bit further.  At least put an 'and' between the two.